src/main/java/school/lemon/changerequest/java/DemoMatrix.java

```diff
@@ -0,0 +1,30 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 12.12.16.
+ */
+public class DemoMatrix {
+    public static void main(String[] args) {
+
+        double[][] matrix_A = {{4., 2.}, {9., 0.}};
```

Comment:
You should always follow Java Code Conventions for naming variables, methods, classes, etc.
Don't use underscores in names. If you already has `matrixA`, name matrixA array as `matrixAarray`.